### PR TITLE
Use config to specify type of meeting doc

### PR DESCRIPTION
**Pull request by mlippert**
## Description
Add a new type of meeting document, a static url, and add supporting configuration settings to specify the type of meeting doc and the url for the static url doc.

In addtion a few other things were addressed:
- some prop type errors that had been showing up in the console for a while have been fixed.
- the timeout for requests to riffdata was increased from the default 5s to 10s
- more vestiges of the lti state were removed
- a configuration setting for whether to show the post meeting survey in the dashboard overlay was added.

## Motivation and context
The said-oxford CyberFutures course simulation is going to use this grant thornton meeting configuration, but needs to show a particular url as the meeting doc. In addtion it is unclear if the post meeting survey overlay will still be needed as the survey will definitely not be a qualtrics survey.

## How has this been tested?
Tested on a test instance in AWS

## Types of changes
- ✅ New feature (non-breaking change which adds functionality)

## Checklist:
- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**

### CODE REVIEW

File: src/redux/selectors/config.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+/* ******************************************************************************
+ * getWebRtcDebug                                                          */ /**
+ *
+ * Get the setting which determines if webrtc debug logging should be enabled.
+ *
+ * @returns {boolean} true if webrtc debugging should be enabled.
+ */
+function getWebRtcDebug() {
+  return !!configSettings.webrtc_debug;
```
Comment (jordanreedie) [inline src/redux/selectors/config.js]: I thought we preferred `Boolean(configSettings.webrtc_debug)` to `!!`?

Comment (mlippert) [reply src/redux/selectors/config.js]: I believe your are correct sir! I totally didn't remember that, and I think I was just copying the existing code. I'm glad you pointed this out before I merged the code.

**Comment by mlippert** [comment]
Yes having the single url in the config is limiting, but it matches the specific use case for Oxford.

Allowing the person starting a meeting to specify the doc url would be a nice enhancement, although there are other constraints I'm not sure how we would convey, namely some sites don't allow themselves to be in an iframe. (I saw that w/ the default url I put in there, stackoverflow).
Still, that could just end up being another meeting docType.

**Comment by mlippert** [comment]
force pushed w/ squashed change to use `Boolean` instead of `!!`

**Pull request merged**